HLActionItem[] hl_actions
---
bool success
string message